### PR TITLE
Use UTC times

### PR DESCRIPTION
**Pull request by tleraj**
The purpose of this PR is to set every date appearing in the app to UTC

### CODE REVIEW

File: packages/website/src/routes/ReefRoutes/Reef/ReefInfo/index.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ const ReefNavBar = ({
             {lastSurvey && (
               <Grid item>
                 <Typography variant="subtitle1">{`Last surveyed: ${moment(
-                  lastSurvey
+                  new Date(lastSurvey).toISOString()
```
Comment (ericboucher) [inline packages/website/src/routes/ReefRoutes/Reef/ReefInfo/index.tsx:47]: seems weird to use moment AND Date, no?

Comment (tleraj) [reply packages/website/src/routes/ReefRoutes/Reef/ReefInfo/index.tsx:47]: The use of moment is only for the display format (e.g "MM/DD/YYYY").

Comment (ericboucher) [reply packages/website/src/routes/ReefRoutes/Reef/ReefInfo/index.tsx:47]: Seems like an overkill. How about creating a simple utility function like this? https://stackoverflow.com/a/11172083

Comment (tleraj) [reply packages/website/src/routes/ReefRoutes/Reef/ReefInfo/index.tsx:47]: We could also get rid of the `Date` like this:

```
moment(moment(lastSurvey).toISOString()).format("MMM DD[,] YYYY")
```

Does that sound good?

**Comment by ericboucher** [comment]
We will use local time instead

**Pull request closed without merging**